email_notification/helpers/email_validator.py

```diff
@@ -1,48 +1,57 @@
 from __future__ import annotations
 
-import os
 import re
 from dataclasses import asdict
 from typing import TYPE_CHECKING
 
-from django.conf import settings
-from django.core.exceptions import ValidationError
+from jinja2 import PackageLoader
 from jinja2.exceptions import TemplateError
 from jinja2.sandbox import SandboxedEnvironment
 
 from email_notification.exceptions import EmailTemplateValidationError
+from email_notification.helpers.email_translations import EMAIL_TRANSLATIONS
 from email_notification.templatetags import format_currency
 
 if TYPE_CHECKING:
-    from django.core.files.uploadedfile import InMemoryUploadedFile
-
     from email_notification.helpers.email_builder_base import BaseEmailContext
 
+
 EMAIL_TEMPLATE_SUPPORTED_EXPRESSIONS = ["if", "elif", "else", "endif"]
 FILTERS_MAP = {"currency": format_currency}
 
 
 class EmailTemplateValidator:
     """Helper class used to validate and safely render email templates."""
```

Comment:
This can be left for later, but seems like this class is not just a validator, it's also a render. I would separate this into two classes.

Reply:
Yeah I thought about that also, but didn't split it yet, as both require `SandboxedEnvironment` and its config.
Maybe just renaming this class would be enough? (And then using `EmailTemplateRenderer` to validate also)